packages/editor/src/components/ComponentForm/JsonSchemaForm/widgets/ExpressionWidget.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useEffect, useState } from 'react';
+import {
+  toNumber,
+  isString,
+  isNumber,
+  isBoolean,
+  isFunction,
+  isObject,
+  isUndefined,
+  isNull,
+} from 'lodash-es';
+import { FieldProps } from '../fields';
+import { ExpressionEditor } from '../../../CodeEditor';
+
+type Props = FieldProps;
+
+// FIXME: move into a new package and share with runtime?
```

Comment:
OK. It is necessary. Actually many util functions are duplicated in runtime and editor.